src/components/Sections/SectionNovelty/index.scss

```diff
@@ -0,0 +1,42 @@
+.novelty {
+    margin-top: 90px;
```

Comment:
If this is the common spacing for all sections, then we could create a separate class for that spacing and use it in the HTML wherever needed.